ddtrace/vendor/debtcollector/removals.py

```diff
@@ -319,16 +315,44 @@ def removed_module(module, replacement=None, message=None,
         module_name = module
     else:
         _qual, type_name = _utils.get_qualified_name(type(module))
-        raise TypeError("Unexpected module type '%s' (expected string or"
-                        " module type only)" % type_name)
+        raise TypeError("Unexpected module type '%s' (expected string or" " module type only)" % type_name)
     prefix = "The '%s' module usage is deprecated" % module_name
     if replacement:
         postfix = ", please use %s instead" % replacement
     else:
         postfix = None
-    out_message = _utils.generate_message(prefix,
-                                          postfix=postfix, message=message,
-                                          version=version,
-                                          removal_version=removal_version)
-    _utils.deprecation(out_message,
-                       stacklevel=stacklevel, category=category)
+    out_message = _utils.generate_message(
+        prefix, postfix=postfix, message=message, version=version, removal_version=removal_version
+    )
+    _utils.deprecation(out_message, stacklevel=stacklevel, category=category)
+
+
+def removed_constant(
```

Comment:
is this going to show the deprecation warning always on startup/import of `ddtrace.constants`?

if so, it would be better to only show the message if someone imports/uses `ddtrace.constants.ERROR_MSG`

I cannot remember how we did that in the past 🤔 

I think @P403n1x87 might know (I think they did it before)

Reply:
I think we did it with PEP 562

Reply:
@brettlangdon Yup, I did something similar to PEP 562 that can be used to deprecate a variable / constant. should be all set